JustTweak/Classes/DTOs/Tweak.swift

```diff
@@ -5,12 +5,12 @@
 
 import Foundation
 
-public struct Tweak {
+public struct Tweak: Hashable {
     
     public let feature: String
     public let variable: String
     
-    public let value: TweakValue
+    public let value: AnyTweakValue
```

Comment:
After thinking about it a little bit. Maybe could be of a generic type T, so in that case Tweak itself would be generic. But then instead of having ‘AnyTweakValue’, we could maybe get rid of that and replace it with ‘AnyTweak’. Which would be the type erased version of ‘Tweak’

Reply:
After trying it a bit, changing `Tweak` to `Tweak<T: TweakValue>` would require more changes. And `AnyTweakValue` would still be needed. So in that case we would need to use `Tweak<AnyTweakValue>` in a bunch of places